### DISCUSSION TITLE
[ FF7 ] Crash after Jenova fight in the boat

### DISCUSSION THREAD

**Discussion by beb1senate** [julianxhokaxhiu/FFNx#383] [General]
Since 1.10.1.159 somrthing like that, always crash at the same time, before catching ifrit materia after jenova boss fight.
Before that crash I had crash before the fight,1.10.1.159+ don't like, possible to download previous version ? A window appear and propose me to go to FAQ...
I have save if you want. I will try with 1.10.1.0

**Comment by beb1senate** [comment]
even with 1.10..1.169

**Comment by beb1senate** [comment]
![image](https://user-images.githubusercontent.com/63313697/153573111-63767aed-3aa3-49f7-b335-00064da250da.png)
![image](https://user-images.githubusercontent.com/63313697/153573210-2f00592c-9e2f-4b57-83fb-70077cbb4d6d.png)

**Comment by beb1senate** [comment]
ok with 1.10.1.0 but it's too fast how to reduce the game speed ?

**Reply by julianxhokaxhiu** [reply]
Update to the latest canary

**Reply by julianxhokaxhiu** [reply]
Also please use thread to reply, do not create a new comment everytime. Thanks.
